distribution/client/src/client.ts

```diff
@@ -93,13 +93,13 @@ export default class Client {
 
     this.runUpdates();
 
-    this.healthChecker.check().then((state) => {
-      if (state.healthy) {
-        UI.publish('Jenkins appears to be online', { log: 'info' });
-        Storage.removeBootingFlag();
-      } else {
-        UI.publish('Jenkins appears to be in an unhealthy state!', { log: 'error' });
-      }
+    this.healthChecker.check().then(() => {
+      UI.publish('Jenkins appears to be online', { log: 'info' });
+      Storage.removeBootingFlag();
+    }).catch((error) => {
+      const errorMessage = 'Jenkins appears to be in an unhealthy state (startup auto-healthcheck)!'
+      logger.error(errorMessage);
+      UI.publish(errorMessage, { log: 'error' });
```

Comment:
Lines 101 and 102 are redundant. Passing the `{ log: 'error' }` to `UI.publish` means that it will perform a `logger.error.` underneath.